CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - [webpack] Source maps for the frontend renamed from `webpack://[namespace]/[resource-filename]...`
   to `webpack:///[resource-path]?[loaders]` where `resource-path` is the path to the file relative
   to your application package's root.
+- [plugin] `WebviewThemeDataProvider.getActiveTheme()` returns `Theme` instead of `WebviewThemeType` and `WebviewThemeData` interface modified [#10493](https://github.com/eclipse-theia/theia/pull/10493)
```

Comment:
Suggestion: We could improve the wording in here to make it easier to grasp what changed in `WebviewThemeData` (we don't need to explicitely state the added name property):

```suggestion
- [plugin] changed return type of `WebviewThemeDataProvider.getActiveTheme()` to `Theme` instead of `WebviewThemeType` [#10493](https://github.com/eclipse-theia/theia/pull/10493)
- [plugin] renamed `WebviewThemeData.activeTheme` to `activeThemeType` [#10493](https://github.com/eclipse-theia/theia/pull/10493)
```

Reply:
@msujew thanks, done!